tests/test_setting_name.py

```diff
@@ -0,0 +1,106 @@
+"""Tests autosetting of file names
+"""
+import os
+from pathlib import Path
+
+import pytest
+
+from ecl2df import ecl2csv
+from ecl2df.common import find_name_components, make_output_name
+from ecl2df.constants import SUBMODULES
+
+TESTDIR = Path(__file__).absolute().parent
+REEK = str(TESTDIR / "data/reek/eclipse/model/2_R001_REEK-0.DATA")
+
+
+@pytest.fixture(name="name_args", scope="session")
+def _fixture_arg_names():
+    """Return the name variables to test against
+
+    Returns:
+        tuple: name, submodule name, and args dict
+    """
+    _keep_name = "TESTING"
+    _subcommand = "summary"
+    _args = {
+        "DATAFILE": f"{_keep_name}-123.DATA",
+        "subcommand": _subcommand,
+        "output": None,
+    }
+
+    return _keep_name, _subcommand, _args
+
+
+def test_find_name_components(name_args):
+    """Test function get_name_from_args
+
+    Args:
+        name_args (tuple): name, submodule name, and args dict
+    """
+    keep_name, subcommand, args = name_args
+    name, tagname = find_name_components(args)
+    assert name == keep_name, f"Name is {name}, but should be {keep_name}"
+    assert tagname == subcommand, f"tagname is {tagname}, but should be {subcommand}"
+
+
+def test_set_name_from_eclbase():
+    """Test function set_name_from_args when no suffix supplied
+
+    Args:
+        name_args (tuple): name, submodule name, and args dict
+    """
+    args = {
+        "subcommand": "mycommand",
+        "DATAFILE": str(REEK),
+        "output": None,
+    }
+
+    correct_name = "2_R001_REEK"
+    correct_tag = "mycommand"
+    name, tag = find_name_components(args)
+    assert name == correct_name, f"Name is {name}, should be {correct_name}"
+    assert tag == correct_tag, f"Tag is {tag}, should be {correct_tag}"
+
+
+def test_set_name_from_args(name_args):
+    """Test function set_name_from_args
```

Comment:
Seems to not do this, it is testing `make_output_name`. Should the test name and the docstring reflect better what it is testing?

Reply:
Well spotted, renamed function and forgot to rename in docstring.. Will fix..